third_party/kmod_eret_hvc_smc/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Builds the kernel module for speculation over ERET, HVC and SMC.
+
+# Check if we should skip building the module.
+if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
Two of these conditions -- "is Linux" and "is crosscompiling" -- are universal and should go back into the function definition.

Reply:
Fixed.